src/messages/components/EmojiButton.vue

```diff
@@ -22,7 +22,13 @@ function getEmojiElement (name) {
   const cached = EMOJI_CACHE[name]
   if (cached) return EMOJI_CACHE[name].cloneNode(true)
   const container = document.createElement('div')
-  container.innerHTML = twemoji.parse(emojiList[name])
+  if (Object.keys(emojiList).includes(name)) {
+    container.innerHTML = twemoji.parse(emojiList[name])
+  }
+  // On error, we display the 'carrot' emoji
```

Comment:
You can remove this exception again, I'll fix it in the backend before merging this PR.

Reply:
Will do!